lib/models/filters.dart

```diff
@@ -77,7 +77,7 @@ class ContestFilter {
           _maxDuration = const Duration(days: 31);
           break;
         default:
-          return true;
+          _maxDuration = const Duration(days: 365);
```

Comment:
Does this mean that infinity = 365 days?

Reply:
I thought no contest information is released before a year. That's Why I have used 365 days duration. I have increased this duration to 10^5 in my later commit. Hope that would be fine.